src/DashboardService.Models.Dto/Requests/Board/PatchBoardRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Board;
+
+public class PatchBoardRequest
+{
+  public Guid Id { get; set; }
+
+  public Guid ProjectId { get; set; }
+
+  public string Name { get; set; }
+
+  public bool IsActive { get; set; }
+
+  public Guid ModifiedBy { get; set; }
```

Comment:
this is not sent in the request, this is set already when handling the request

Reply:
Remove, but wasn't add PatchDbBoardMapper

Reply:
was add